packages/Card/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# @igloo-ui/card
+
+## 0.0.2
+### Patch Changes
+
+- fa1a096: ## Card component
+  
+  Cards are used a the background that contains elements of a parent ensemble.
```

Comment:
I see this sentence and I don't understand what we are trying to explain here.

Maybe (if I understand correctly) this version could be clearer:
> Cards are used as a visual container for a sub-group of elements in a parent container.

Reply:
Je prend en note et ouvre une issue avec ton commentaire.